src/status_im2/contexts/chat/menus/pinned_messages/view.cljs

```diff
@@ -8,7 +8,8 @@
             [status-im2.contexts.chat.menus.pinned-messages.style :as style]
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]
-            [react-native.gesture :as gesture]))
+            [react-native.gesture :as gesture]
+            [quo.design-system.spacing :as spacing]))
```

Comment:
hmm, we should not be using anything from `quo` namespace

Reply:
but we do have a spacings in foundation which is not in our codebase for some reason 🙃 https://www.figma.com/file/v98g9ZiaSHYUdKWrbFg9eM/Foundations?type=design&node-id=624-350&mode=design&t=xed0yo4wQPZIiuWa-0